homeassistant/components/airthings/sensor.py

```diff
@@ -156,4 +156,4 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the value reported by the sensor."""
-        return self.coordinator.data[self._id].sensors[self.entity_description.key]  # type: ignore[no-any-return]
+        return self.coordinator.data[self._id].sensors.get(self.entity_description.key)  # type: ignore[no-any-return]
```

Comment:
While I get this fixes it, I do not think this is the right approach.

> the key (sensor type) might not exist anymore.

Doesn't this influence its availability instead? Because, we normally in these cases, implement the `available` property method to handle this.